Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -469,7 +469,11 @@ public final class PackageBuilder {
                 if subpath == "" || subpath == "." {
                     return packagePath
                 }
-                let path = packagePath.appending(RelativePath(subpath))
+
+                // Make sure the target path is valid.
+                try AbsolutePath.validate(path: subpath)
+                let path = AbsolutePath(subpath, relativeTo: packagePath)
```

Comment:
Wait, I don't think we want this. We don't allow absolute path in `path` property.